adi/ltc2672.py

```diff
@@ -0,0 +1,219 @@
+# Copyright (C) 2024 Analog Devices, Inc.
+#
+# SPDX short identifier: ADIBSD
+
+from decimal import Decimal
+
+from adi.attribute import attribute
+from adi.context_manager import context_manager
+
+
+class ltc2672(context_manager, attribute):
+    """LTC2672 DAC"""
+
+    _complex_data = False
+    channel = []
```

Comment:
**channel** should not be a class property or it needs to be reset within the constructor. This creates an issue when you have multiple instances of the class. channel will keep growing